hasura/metadata/tables.yaml

```diff
@@ -315,7 +315,6 @@
       - timezone
       - username
       filter: {}
-      allow_aggregations: true
```

Comment:
I don't think you want to do this? Looks like a bad merge

Reply:
Fixed it :)